src/System.Private.ServiceModel/tools/scripts/WCFCustomCleanup.cmd

```diff
@@ -0,0 +1,36 @@
+@if "%_echo%" neq "on" echo off
```

Comment:
My personal opinion is that the dev should be able to run StartWcfService from a clean enlistment and have it build and start.  That is how StartBridge worked.  As far as allowing the dev to call other scripts by hand, I'd support that only if necessary to get a good experience with VS.  For example, can I just do StartWcfService and then go run tests from VS?  Or would I also need to run other scripts?  My most common scenario is:
```
   build /p:SkipTests=true
   StartBridge (or StartWcfService)
```
then I open VS and select the test I want run.